test/config/integration/certs/servercert_info.h

```diff
@@ -0,0 +1,5 @@
+// NOLINT(namespace-envoy)
+constexpr char TEST_SERVER_CERT_HASH[] = "DC:E2:2B:65:90:43:9A:36:1C:8E:6D:CA:42:8A:8C:37:C7:A1:77:"
+                                         "00:5B:C1:3E:33:8A:B9:2D:04:2C:B1:3F:0A";
+constexpr char TEST_SERVER_CERT_NOT_BEFORE[] = "Apr 7 16:46:35 2022 GMT";
+constexpr char TEST_SERVER_CERT_NOT_AFTER[] = "Apr 6 16:46:35 2024 GMT";
```

Comment:
Can you add comments on where this came from and how to regenerate it if needed?

Reply:
All are generated by `certs.h`, refer to https://github.com/envoyproxy/envoy/blob/main/test/config/integration/certs/README.md, or I can update that script and add comments during generating.